### PR TITLE
Add mute/unmute bot methods

### PR DESCRIPTION
**Pull request by windkomo**

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -194,6 +194,26 @@ class Bot {
...
   }
 
+  /**
+   * Set isMuted to true
+   * @async
+   * @private
+   * @returns {void}
+   */
+  async mute() {
+    return this.brain.setValue('isMuted', true);
```
Comment (yangeorget) [inline src/bot.js]: what about _isMuted?

**Comment by yangeorget** [comment]
what about using _ to prefix Botfuel reserved entries?

**Comment by windkomo** [comment]
@yangeorget Is it really a Botfuel reserved entry?
It's meant to be used by developers to mute/unmute their bot, so I think it should be "public" in the brain?

**Comment by windkomo** [comment]
Closing this because we agree that the feature people look for is handing over to an operator rather than muting.

This feature can still be implemented easily at bot level using the brain and a middleware.

**Pull request closed without merging**